### DISCUSSION TITLE
主题很漂亮 希望增加一些功能

### DISCUSSION THREAD

**Discussion by blblank** [Tokinx/Wing#111] [讨论]
1、显示文章目录
2、显示文章路径
3、评论添加验证码
4、热门文章、最新文章、最新评论
5、无刷新搜索
6、云标签
7、博主评论增加标识
8、文章摘要
9、回到顶部、到底部、悬浮按钮

**Comment by Tokinx** [comment]
1、有计划
2、无计划
3、无计划
4、小工具可以添加这些
5、有计划
6、计划添加到文章归档页面
7、有计划
8、无计划
9、计划添加滚动到顶部、评论区功能

以上计划建立在新电脑到手之后再排期开发
2333，可能陆续要排到8月
